src/widgets/main/solution/section/style.module.scss

```
@@ -16,6 +16,23 @@
 	white-space: pre-line;
 }
 
+.solution_subtitle {
+	@include token.body_medium_medium;
+	font-size: token.$font_size_18;
```

Stylelint (17.13.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)